modules/persistent-storage-csi-azure-file-cross-sub-dynamic-provisioning-procedure.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent_storage-csi-azure-file.adoc
+//
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-azure-file-cross-sub-dynamic-provisioning-procedure_{context}"]
+= Dynamic provisioning across subscriptions for Azure File
+
+.Prerequisites
+* Installed Azure cluster with the service principal or managed identity as Azure identity
+
+* Access to another subscription in the tenant with the cluster  
+
+* Logged in to the Azure CLI
+
+.Procedure
+To use Azure File dynamic provisioning across subscriptions:
+
+. Record the Azure identity (service principal or managed identity) by running the following applicable commands. The Azure identity is needed in the next step:
```

Comment:
Do we need (service principal or managed identity) here, as it is in the prereqs?

```suggestion
. Record the Azure identity by running the following applicable commands. The Azure identity is needed in the next step:
```